### ISSUE TITLE
Hotkeys stopped working after most recent update

### ISSUE THREAD

**Issue by DRSXS** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1422]
**OS:**
Windows 10

**Issue Descriptions:**
Without changing anything the Hotkeys I had set for volume up and down (ctrl + up/down) stopped working after the most recent update.

**Comment by MarshallOfSound** [comment]
@DRSXS Are they still set correctly in your settings?

**Comment by DRSXS** [comment]
Yes, I checked that. Still set https://gyazo.com/84ca9aecc529186b7b049b2fd94aefee

**Comment by Alipoodle** [comment]
Same problem here! I also have them still "set".
However Media Keys still work for Pause/Player Next and Previous however these are not set in the Desktop Settings

**Comment by jwmann** [comment]
I have the same issue on OS X 10.11.4 -- most recent version
Never changed my global hotkeys but can confirm they are still set.

Regular keyboard media keys still work though.

**Comment by MarshallOfSound** [comment]
@Alipoodle @jwmann Can you try "resetting" them.

I.e. Unset and then set them again

**Comment by jwmann** [comment]
@MarshallOfSound 
I tried _changing_ the key to something else and then back and it didn't affect anything.

Also, I'm not sure how you 'unset' a keybinding though. Delete/Backspace doesn't seem to unset anything.

**Comment by MarshallOfSound** [comment]
@jwmann It literally tells you how at the top of the settings page 😆 

> Use the "Escape" key to reset the hotkey

Regardless, can you try restarting your computer and trying again.  Sometimes hotkeys get lost to oblivion.

Also can you try setting it to a completely random hotkey you haven't used before and trying again.

**Comment by jwmann** [comment]
@MarshallOfSound Wow, sorry I'm blind apparently.

I tried _unsetting_ the keys and then assigned it a new shortcut I never used before and still nothing. (original shortcuts don't work either)

I'll try restarting my computer to see if it helps (seems to fix most things haha)

**Comment by jwmann** [comment]
@MarshallOfSound Okay so I restarted my computer. Performed the same troubleshooting techniques as before. Hotkeys still don't work.

**Comment by MarshallOfSound** [comment]
omigod

I just looked at that screenshot @DRSXS 

@jwmann Are the shortcuts you are trying to set for increase and decrease volume?

**Comment by DRSXS** [comment]
@MarshallOfSound Alright, I did some testing. 

All hotkeys other than volume up and down seem to work fine. Not all key combinations can be used but I suppose that is intended behaviour as to not conflict with other hotkeys (e.g. from Windows). 

I tried setting a different hotkey, e.g. previous track/next track to the same key combination and it worked. So I suspect it directly has to do with the volume controls and not with a specific key combination.
